verticalseekbarlib/src/main/java/opencraft/com/verticalseekbarlib/VerticalSeekBar.java

```diff
@@ -37,88 +30,83 @@ public VerticalSeekBar(Context context, @Nullable AttributeSet attrs) {
         super(context, attrs);
         inflater = LayoutInflater.from(context);
         init();
-        loadAttrs(context, attrs);
+        loadCustomAttrs(context, attrs);
     }
 
     private void init() {
         layoutView = inflater.inflate(R.layout.vertical_seek_bar, this, true);
         verticalSeekBarThumb = layoutView.findViewById(R.id.agiplan_custom_seekbar_thumb);
         verticalSeekBarBackground = layoutView.findViewById(R.id.agiplan_custom_seekbar_background);
-
         addThumbTouchListener();
     }
 
-    private void loadAttrs(Context context, AttributeSet attrs) {
+    private void loadCustomAttrs(Context context, AttributeSet attrs) {
         TypedArray attributes = context.getTheme().obtainStyledAttributes(
                 attrs,
                 R.styleable.VerticalSeekBar,
                 0, 0);
+        loadBackgroundAttr(attributes);
+        loadThumbAttr(attributes);
+        loadValueAttr(attributes);
+        loadStepAttr(attributes);
+        setupBackgroundMarginTopAttr(attributes);
+    }
 
+    private void loadValueAttr(TypedArray attributes) {
+        valor = attributes.getInteger(R.styleable.VerticalSeekBar_seekbar_value, 500);
+    }
 
+    private void loadStepAttr(TypedArray attributes) {
+        step = attributes.getInteger(R.styleable.VerticalSeekBar_seekbar_step, 25);
```

Comment:
What is 25? Can you put that in a constant?

Reply:
same as above